src/front-end/src/App.js

```diff
@@ -28,6 +29,7 @@ function App() {
         </Header>
         <Content style={{ padding: '0 50px' }}>
           <Switch>
+            <Route path="/ForgotPassword" component={ForgotPassword} />
             <Route path="/Auth" component={Auth} />
             {/* <Route path="/MentorHome" component={MentorHome} /> */}
```

Comment:
Minor: you may not have added this comment, but do you mind removing it too, while you're at it? Thanks!

Reply:
done